lib/pagseguro/payment_releases.rb

```diff
@@ -10,12 +10,15 @@ def initialize
       @payments = []
     end
 
+    # Adds payment to payment list.
     def <<(payment)
       payment = ensure_type(PaymentRelease, payment)
 
       @payments << payment unless @payments.include? payment
     end
 
+    # Verify if a payment is already included to payment list.
+    # Returns Boolean.
```

Comment:
Looks ok, I was wondering why is not the YARD format, so when generating the documentation could not generate correctly